vpr/src/base/read_options.cpp

```diff
@@ -1310,6 +1310,11 @@ argparse::ArgumentParser create_arg_parser(const std::string& prog_name, t_optio
         .action(argparse::Action::STORE_TRUE)
         .default_value("off");
 
+    stage_grp.add_argument<bool, ParseOnOff>(args.do_legalize, "--legalize")
+        .help("Legalize a flat placement")
```

Comment:
Good to expand this a bit. Briefly explain what a flat placement is (atom-level placement that controls packing and placement).